server/src/test/java/io/crate/testing/Asserts.java

```diff
@@ -40,4 +42,14 @@ public static void assertThrows(Executable executable, Matcher<? super Throwable
             assertThat(t, matcher);
         }
     }
+
+    public static void assertThrows(Executable executable, Class<? extends Throwable> type, String subString) {
```

Comment:
Isn't this the same as `org.junit.jupiter.api.Assertions.assertThrows` ?

Reply:
On the methods I know of/found, the given message is used as a prefix for a failure output instead of matching the thrown exception message. Or do other `assertThrows` implementation exists which I'm not aware of?

Reply:
Oh right :+1: